convert/terratag_locals.go

```diff
@@ -0,0 +1,71 @@
+package convert
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/hashicorp/hcl/v2/hclwrite"
+)
+
+var localsRegex = regexp.MustCompile(`"([^"]*)"[ \t]*=[ \t]*"([^"]*)"`)
+
+type Locals map[string]string
+
+func decodeTerratagLocals(locals Locals, s string) error {
```

Comment:
Made some naming changes to help clarify this only applies to the terratag locals.